src/NHibernate.Test/Linq/WhereTests.cs

```diff
@@ -648,6 +649,9 @@ where sheet.Users.Contains(user)
 		[Test]
 		public void TimesheetsWithEnumerableContainsOnSelect()
 		{
+			if (Dialect is MsSqlCeDialect)
```

Comment:
Why?

Reply:
fixes SqlServerCe test for https://github.com/nhibernate/nhibernate-core/pull/3271

See:  https://teamcity.jetbrains.com/buildConfiguration/bt1175/4111194?buildTab=tests&expandedTest=build%3A%28id%3A4111194%29%2Cid%3A29811

> System.IndexOutOfRangeException : Index was outside the bounds of the array.
 at System.Data.SqlServerCe.SqlCeCommand.GetQueryParameters(IntPtr pQpCommand)
     at System.Data.SqlServerCe.SqlCeCommand.CreateDataBindings()
     at System.Data.SqlServerCe.SqlCeCommand.CompileQueryPlan()